pkg/controller/compliancescan/compliancescan_controller_test.go

```diff
@@ -205,25 +233,60 @@ var _ = Describe("Testing compliancescan controller phases", func() {
 		BeforeEach(func() {
 			// Set state to RUNNING
 			compliancescaninstance.Status.Phase = compv1alpha1.PhaseLaunching
+			compliancescaninstance.Status.StartTimestamp = &metav1.Time{Time: time.Now()}
 			err := reconciler.Client.Status().Update(context.TODO(), compliancescaninstance)
 			Expect(err).To(BeNil())
 		})
-		It("should create PVC and stay on the same phase", func() {
-			result, err := reconciler.phaseLaunchingHandler(handler, logger)
-			Expect(result).ToNot(BeNil())
-			Expect(err).To(BeNil())
-			Expect(compliancescaninstance.Status.Phase).To(Equal(compv1alpha1.PhaseLaunching))
+		Context("With no PVC", func() {
+			It("should create PVC and stay on the same phase", func() {
+				result, err := reconciler.phaseLaunchingHandler(handler, logger)
+				Expect(result).ToNot(BeNil())
+				Expect(err).To(BeNil())
+				Expect(compliancescaninstance.Status.Phase).To(Equal(compv1alpha1.PhaseLaunching))
 
-			// We should have scheduled a pod per node
-			scan := &compv1alpha1.ComplianceScan{}
-			key := types.NamespacedName{
-				Name:      compliancescaninstance.Name,
-				Namespace: compliancescaninstance.Namespace,
-			}
-			err = reconciler.Client.Get(context.TODO(), key, scan)
-			Expect(err).To(BeNil())
-			Expect(scan.Status.ResultsStorage.Name).To(Equal(getPVCForScanName(key.Name)))
+				// We should have scheduled a pod per node
+				scan := &compv1alpha1.ComplianceScan{}
+				key := types.NamespacedName{
+					Name:      compliancescaninstance.Name,
+					Namespace: compliancescaninstance.Namespace,
+				}
+				err = reconciler.Client.Get(context.TODO(), key, scan)
+				Expect(err).To(BeNil())
+				Expect(scan.Status.ResultsStorage.Name).To(Equal(getPVCForScanName(key.Name)))
+			})
 		})
+
+		Context("with the PVC set and no Kubelet ConfigMap", func() {
```

Comment:
nit: Should `with` here be capitalized to be consistent with the context above?

Reply:
Do we want a variant of this test for when the kubelet config exists and we clean it up?